dashboard/client/src/common/ProfilingLink.tsx

```diff
@@ -19,14 +20,14 @@ export const CpuProfilingLink = ({
   }
 
   return (
-    <a
```

Comment:
nit: Also think there's more potential usages/refactor-to-`<Link>` of `<a />` in our code